airbyte-integrations/connectors/source-yotpo/source_yotpo/manifest.yaml

```diff
@@ -0,0 +1,221 @@
+version: "0.29.0"
+
+definitions:
+
+  yotpo_stream_requester:
+    type: HttpRequester
+    url_base: "https://api.yotpo.com/v1"
+    http_method: "GET"
+    request_parameters:
+      utoken: "{{ config['access_token'] }}"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['access_token'] }}"
+
+  yotpo_stream_plain_requester:
+    type: HttpRequester
+    url_base: "https://api.yotpo.com"
+    http_method: "GET"
+    request_parameters:
+      utoken: "{{ config['access_token'] }}"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['access_token'] }}"
+
+  privacy_stream_requester:
+    type: HttpRequester
+    url_base: "https://api.yotpo.com/privacy"
+    http_method: "GET"
+    request_parameters:
+      utoken: "{{ config['access_token'] }}"
+      email: "{{ config['email'] }}"
+
+  analytics_stream_requester:
+    type: HttpRequester
+    url_base: "https://api.yotpo.com/analytics/v1"
+    http_method: "GET"
+    request_parameters:
+      utoken: "{{ config['access_token'] }}"
+
+  reviews_stream:
+    type: DeclarativeStream
+    retriever:
+      type: SimpleRetriever
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path: ["reviews"]
+      paginator:
+        type: "DefaultPaginator"
+        page_size_option:
+          type: "RequestOption"
+          inject_into: "request_parameter"
+          field_name: "per_page"
+        pagination_strategy:
+          type: "PageIncrement"
+          page_size: 5
```

Comment:
Where did you find this information in Yopto docs?

Reply:
It was incorrect, Corrected :)

Reply:
Resolved